layers/+window-management/spacemacs-layouts/funcs.el

```diff
@@ -127,11 +150,20 @@ perspectives does."
             :action (lambda (project)
                       (let ((persp-reset-windows-on-nil-window-conf t))
                         (persp-switch project)
+                        (push project layouts-projectile)
                         (let ((projectile-completion-system 'ivy))
-                          (projectile-switch-project-by-name project))))
-            )
+                          (projectile-switch-project-by-name project))))))
 
-  )
+(defun spacemacs/ivy-persp-switch-open-projects (arg)
```

Comment:
The arg is needed as long as we receive an interactive argument (prefix argument), so i left it there. I'm documenting the functions.